### PR TITLE
🌐 Add Traditional Chinese translation for `docs/zh-hant/docs/fastapi-cli.md`

### PR DESCRIPTION
**Pull request by yychanlee**

### CODE REVIEW

File: docs/zh-hant/docs/fastapi-cli.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# FastAPI CLI
+
+**FastAPI CLI** 是一個命令行程式，能用來運行你的 FastAPI 應用程式、管理你的 FastAPI 專案等。
```
Comment (mattwang44) [inline docs/zh-hant/docs/fastapi-cli.md:3]:
```suggestion
**FastAPI CLI** 是一個命令列程式，能用來運行你的 FastAPI 應用程式、管理你的 FastAPI 專案等。
```

File: docs/zh-hant/docs/fastapi-cli.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+<font color="#4E9A06">INFO</font>:     Started server process [<font color="#06989A">2265873</font>]
+<font color="#4E9A06">INFO</font>:     Waiting for application startup.
+<font color="#4E9A06">INFO</font>:     Application startup complete.
+```
+
+</div>
+
+`fastapi` 命令行程式就是 **FastAPI CLI**。
+
+FastAPI CLI 接收你的 Python 程式路徑（例如 `main.py`），並自動檢測 FastAPI 實例（通常命名為 `app`），確定正確的匯入流程，然後運行該應用。
```
Comment (mattwang44) [inline docs/zh-hant/docs/fastapi-cli.md:59]:
```suggestion
FastAPI CLI 接收你的 Python 程式路徑（例如 `main.py`），並自動檢測 FastAPI 實例（通常命名為 `app`），確定正確的引入模組流程，然後運行該應用程式。
```

File: docs/zh-hant/docs/fastapi-cli.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+執行 `fastapi dev` 會啟動開發模式。
+
+預設情況下，**auto-reload** 功能是啟用的，當你對程式碼進行修改時，伺服器會自動重新載入。這會消耗較多資源，並且可能比禁用時更不穩定。因此，你應該只在開發環境中使用此功能。它也會在 IP 位址 `127.0.0.1` 上監聽，這是用於你的機器與自身通信的 IP 位址（`localhost`）。
+
+## `fastapi run`
+
+執行 `fastapi run` 會以生產模式啟動 FastAPI。
+
+預設情況下，**auto-reload** 功能是禁用的。它也會在 IP 位址 `0.0.0.0` 上監聽，表示會監聽所有可用的 IP 地址，這樣任何能與該機器通信的人都可以公開訪問它。這通常是你在生產環境中運行應用的方式，例如在容器中運行時。
```
Comment (mattwang44) [inline docs/zh-hant/docs/fastapi-cli.md:75]: ref: https://hackmd.io/@l10n-tw/glossaries
```suggestion
預設情況下，**auto-reload** 功能是禁用的。它也會在 IP 位址 `0.0.0.0` 上監聽，表示會監聽所有可用的 IP 地址，這樣任何能與該機器通信的人都可以公開存取它。這通常是你在生產環境中運行應用的方式，例如在容器中運行時。
```

File: docs/zh-hant/docs/fastapi-cli.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+在大多數情況下，你會（也應該）有一個「終止代理」來處理 HTTPS，這取決於你如何部署你的應用程式，你的服務供應商可能會為你做這件事，或者你需要自己設置它。
+
+/// tip
+
+你可以在 [deployment documentation](deployment/index.md){.internal-link target=_blank} 中了解更多相關資訊。
```
Comment (mattwang44) [inline docs/zh-hant/docs/fastapi-cli.md:81]:
```suggestion
你可以在[部署文件](deployment/index.md){.internal-link target=_blank}中了解更多相關資訊。
```

File: docs/zh-hant/docs/fastapi-cli.md (hunk 3 above)
Comment (codingjenny) [reply docs/zh-hant/docs/fastapi-cli.md:75]: Thanks for your reference.

**Comment by mattwang44** [comment]
Note that you can add multiple suggestions into a batch and commit them together
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

File: docs/zh-hant/docs/fastapi-cli.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+在生產環境，你應該使用 `fastapi run` 命令。 🚀
+
+**FastAPI CLI** 內部使用了 <a href="https://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a>，這是一個高效能、適合生產環境的 ASGI 伺服器。 😎
+
+## `fastapi dev`
+
+執行 `fastapi dev` 會啟動開發模式。
+
+預設情況下，**auto-reload** 功能是啟用的，當你對程式碼進行修改時，伺服器會自動重新載入。這會消耗較多資源，並且可能比禁用時更不穩定。因此，你應該只在開發環境中使用此功能。它也會在 IP 位址 `127.0.0.1` 上監聽，這是用於你的機器與自身通信的 IP 位址（`localhost`）。
```
Comment (mattwang44) [inline docs/zh-hant/docs/fastapi-cli.md:69]:
```suggestion
預設情況下，**auto-reload** 功能是啟用的，當你對程式碼進行修改時，伺服器會自動重新載入。這會消耗較多資源，並且可能比禁用時更不穩定。因此，你應該只在開發環境中使用此功能。它也會在 IP 位址 `127.0.0.1` 上監聽，這是用於你的機器與自身通訊的 IP 位址（`localhost`）。
```

File: docs/zh-hant/docs/fastapi-cli.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+## `fastapi run`
+
+執行 `fastapi run` 會以生產模式啟動 FastAPI。
+
+預設情況下，**auto-reload** 功能是禁用的。它也會在 IP 位址 `0.0.0.0` 上監聽，表示會監聽所有可用的 IP 地址，這樣任何能與該機器通信的人都可以公開存取它。這通常是你在生產環境中運行應用程式的方式，例如在容器中運行時。
```
Comment (mattwang44) [inline docs/zh-hant/docs/fastapi-cli.md:75]:
```suggestion
預設情況下，**auto-reload** 功能是禁用的。它也會在 IP 位址 `0.0.0.0` 上監聽，表示會監聽所有可用的 IP 地址，這樣任何能與該機器通訊的人都可以公開存取它。這通常是你在生產環境中運行應用程式的方式，例如在容器中運行時。
```

**Review by mattwang44 — COMMENTED** [review]
the rest LGTM

**Review by mattwang44 — APPROVED** [review]

**Comment by alejsdev** [comment]
Thanks for the contribution! @codingjenny :sparkles:
And thanks for the review @mattwang44 :100:

**Pull request merged**